### DISCUSSION TITLE
Seeed Relay Selection Guide | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#186] [Q&A]
# Seeed Relay Selection Guide | Seeed Studio Wiki

Seeed Relay Selection Guide

https://wiki.seeedstudio.com/Seeed_Relay_Page/

**Comment by AggregatVier** [comment]
Very important to microcontroller projects is to know if the relay module can separate the input signal from the coil voltage. And most useful without having to use off-board level converters or optoisolators. Out of all these relays only one may be suitable but there's no column to confirm.

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.
